sdc/rabbit/exceptions.py

```diff
@@ -0,0 +1,24 @@
+class BadMessageError(Exception):
+    """A bad message is broken in some way that will never be accepted by
+    the endpoing and as such should be rejected (it will still be logged
+    and stored so no data is lost).
```

Comment:
It depends on the `_process` callback, which is defined on a per-service basis. I will amend.